static/sass/_v1_pattern_navigation.scss

```diff
@@ -0,0 +1,181 @@
+.p-navigation {
```

Comment:
We only use that convention upstream to allow people to cherry pick if they want to - why would we need that at site level?